strings-csharp/DeterminingIfStringIsNumericOnly/Tests/IsStringNumberUnitTests.cs

```diff
@@ -0,0 +1,171 @@
+using DeterminingIfStringIsNumericOnly;
+
+namespace Tests;
+
+public class IsStringNumberUnitTests
+{
+    [Test]
+    public void GivenIsNumericWithNewRegex_WhenInputIsIntegerOnly_ThenReturnsTrue()
+    {
+        const string input = "1234567890";
+        var isStringNumber = new IsStringNumber();
```

Comment:
Since this is being done in each test, we could make it a private static readonly member of the class and just reference it from there (I believe this works in NUnit, I am not as familiar with the current state of NUnit - if that is not the case, then we can still add a member variable and use the OneTimeSetup attribute in conjunction with TestFixture to handle the initialization)

Reply:
Extracted them, it's just a habit of mine that I declare everything inside the test (or in an instance method and call it from every test), this way unit tests can run in parallel and I don't have to think about whether the system under test is thread-safe or not. However here it shouldn't be an issue since there is no state in the IsStringNumber class, so I made these static properties